ykman-gui/qml/PivCertificateInfo.qml

```diff
@@ -58,11 +58,11 @@ ColumnLayout {
         })
     }
 
-    function importCertificate(fileUrl) {
+    function importFromFile(fileUrl) {
 
         function handleResponse(resp) {
             if (resp.success) {
-                snackbarSuccess.show("Certificate was imported")
+                snackbarSuccess.show("Certificate/Key was imported")
```

Comment:
and/or key? Is it possible to tell which?

Reply:
It is, but requires a bit more code changes. Could do it in a separate PR?

Reply:
Ok. If you feel like it and it seems worth the effort. :)